### ISSUE TITLE
Incorrect file size being specified with file upload

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero-ios#476]
I'm seeing a bunch of messages like this in the logs:

```
Specified file size incorrect for existing file d8ac607b737f2a961ece85999533e4f1/[…].m4a (59096 != 9)
```

```
Specified file size incorrect for existing file bbc18cd86f342309c571055b0da36f21/[…].pdf.pdf (1509202 != 9)
```

The `9` seems bad. This error is for files that already exist, but if `9` is passed for new files that would likely result in an upload failure.

Not sure if this is a current version of the app. User agent is `"Zotero/1 CFNetwork/1329 Darwin/21.3.0"`.

**Comment by michalrentka** [comment]
The value `9` seems to be the `Not found` issue we were dealing with before. I just wonder why it's being submitted to backend.

**Comment by michalrentka** [comment]
@dstillman which request is this? Is it `POST` to `users/{user_id}/items`?

**Comment by dstillman** [comment]
`POST /users/{userID}/items/{itemKey}/file`

**Comment by michalrentka** [comment]
@dstillman do you have timestamps for those logs? Is it possible they are old requests? Like a few months ago.

**Comment by dstillman** [comment]
And if the `1` in `Zotero/1` is the build number, I guess it was with 1.0.1-1.

**Comment by michalrentka** [comment]
@dstillman this should be fixed by #483 and additional little change.

**Comment by dstillman** [comment]
https://forums.zotero.org/discussion/96453/sync-failed
